tardis/montecarlo/src/test_cmontecarlo.c

```diff
@@ -261,17 +261,17 @@ test_rpacket_doppler_factor(void){
         return res;
 }
 
-double
-test_move_packet(void){
-	double DISTANCE = 1e13;
-        rpacket_t rp;
-        storage_model_t sm;
-        init_rpacket(&rp);
-        init_storage_model(&sm);
-	double res = move_packet(&rp, &sm, DISTANCE);
-        dealloc_storage_model(&sm);
-        return res;
-}
+//double
```

Comment:
Well - I wouldn't for now. Eventually, we want to test move_packet, just in a different way. I'd keep these lines as a reminder.